src/data-table/data-value/valueRenderers.js

```diff
@@ -48,7 +48,13 @@ class DateValue extends PureComponent {
         // Get the locale from the userSettings
         this.context.d2.currentUser.userSettings
             .get('keyUiLocale')
-            .then(uiLocale => this.setState({ uiLocale }));
+            .then(uiLocale => {
+                if (uiLocale === 'in_ID') {
```

Comment:
Please add a comment with a reference to the JIRA ticket so it's possible to follow a breadcrumb trail to the root issue, with a brief explanation of why the hack is here and on which criteria we can remove it.

Reply:
Done. Please have another look @varl

Reply:
Perfect, thanks!